WalletWasabi.Documentation/CodeCoverage.md

```diff
@@ -1,12 +1,35 @@
-Install `altcover` NuGet to the `WalletWasabi.Tests` project and Coverage Glutters extension to vscode.
+# Code coverage (How to)
+
+Wasabi wallet is build using dotnet core. Given there is no a cross-platform Profiling API like the one available on Windows, we 
+use [AltCover](https://github.com/SteveGilham/altcover) package for instrumenting the assemblies and recording the execution
+coverage.
+
+So, first of all we need to install the package in the `WalletWasabi.Tests` project as follow:
+
+```sh
+dotnet add WalletWasabi.Tests/WalletWasabi.Tests.csproj package AltCover
+```
+
+Next, we must build the solution as usual. After that, it is necessary to instrument the assemblies, run the tests and collect coverage:
 
 ```sh
 dotnet build
-dotnet C:\Users\user\.nuget\packages\altcover\5.0.663\tools\netcoreapp2.0\altcover.dll --save --inplace "-i=bin\Debug\netcoreapp2.2"
+
+altcover=$(locate AltCover.dll | tail -1)
+dotnet $altcover --save --inplace "-i=./WalletWasabi.Tests/bin/Debug/netcoreapp2.2"
```

Comment:
I thought we don't need this anymore.

Reply:
I didn't delete it because sometimes you don't need/want to reinstrument the assemblies again. I am thinking in cases where you calcel the test run or some tests fail and you want to re-run them without to recompile and reinstrument everything. But yes I can delete it.